sql/session.go

```diff
@@ -340,7 +370,7 @@ var autoSessionIDs uint32 = 1
 
 // NewBaseSession creates a new empty session.
 func NewBaseSession() Session {
-	return &BaseSession{id: atomic.AddUint32(&autoSessionIDs, 1), config: DefaultSessionConfig(), mu: &sync.RWMutex{}, locks: make(map[string]bool)}
+	return &BaseSession{id: atomic.AddUint32(&autoSessionIDs, 1), config: DefaultSessionConfig(), mu: &sync.RWMutex{}, locks: make(map[string]bool), lastQueryInfo: defaultLastQueryInfo()}
```

Comment:
Probably better to break this up rather than having 5 fields set on one line.

Reply:
Good call